src/plugins/intel_cpu/src/node.h

```diff
@@ -265,6 +295,9 @@ class Node {
 
     bool isInPlace() const;
 
+    virtual bool canBeSkipped() const {
+        return getSelectedPrimitiveDescriptor()->hasZeroInputDims();
+    }
```

Comment:
May be change the name to `isExecutableStatic` and rename the existing `isExecutable` -> `isExecutableDynamic` in the analogy with executeStatic/Dynamic? What do you think?

Reply:
The naming is hard in this case.
My thoughts are:
1) We are trying to perform an optimization, when we completely drop a node from the execution graph, because we know it will be never executed. The good name for such check would be actually "isExecutable", meaning "is executable at all?"
2) We are trying to check, whether the node execution can be skipped during the inference, because it gets zerodim input shapes or maybe it became inplace, when we will have dynamic inplace. This check could have a name "shouldBeSkipped", "mustBeSkipped" or "isDynamicallyExecutable().